src/firmware/boards/robot_stm32h7/freertos.c

```diff
@@ -243,55 +344,11 @@ __weak void io_proto_multicast_startNetworkingTask(void *argument)
     /* Infinite loop */
     for (;;)
     {
-        osDelay(1);
+        osDelay(10000);
```

Comment:
why 10000?

Reply:
this function is unused, not sure why I changed it
removed